README.md

```diff
@@ -26,17 +26,39 @@ In this repository, the processes and features shown in a GitOps process of deve
 
 To be successful in recreating the use cases supported by this pipeline, there are initial steps that should be completed prior to configuring this repository:
 
-- Completion of all pre-requisites and confiuration steps leading to [Feature Development](https://github.com/pingidentity/pipeline-example-platform?tab=readme-ov-file#feature-development) in the example-pipeline-platform.
-- [Docker](https://docs.docker.com/engine/install/) - used to deploy a local sample application.
+- Completion of all pre-requisites and configuration steps leading to [Feature Development](https://github.com/pingidentity/pipeline-example-platform?tab=readme-ov-file#feature-development) from the example-pipeline-platform repository
+- [Docker](https://docs.docker.com/engine/install/) - used to deploy the UI for a sample interface
+- [tflint](https://github.com/terraform-linters/tflint) - for Terraform linting
+- [dvlint](https://github.com/pingidentity/dvlint) - for Davinci flow linting
+- [trivy](https://github.com/aquasecurity/trivy) - for security scanning
+
+> Note - The last three tools are used by the pipeline in Github, and the pipeline will fail if these tests and configuration checks do not pass. Installing these tools locally and running `make devcheck` before committing changes should ensure that the pipeline will pass when changes are pushed.
 
 <!-- TODO - Review Required Permissions-->
 > Note - For PingOne, meeting these requirements means you should have credentials for a worker app residing in the "Administrators" environment that has organization-level scoped roles. For DaVinci, you should have credentials for a user in a non-"Administrators" environment that is part of a group specifically intended to be used by command-line tools or APIs with environment-level scoped roles.
 
+### Development Environment
+
+If you have not created a static development environment in your PingOne account, you can do so in the local copy of the *pipeline-example-platform* repository by running the following commands to instantiate one matching prod and qa:
+
+```bash
+git checkout prod
+git pull origin prod
+git checkout -b dev
+git push origin dev
+```
+
+Capture the environment ID for the development environment for use later.
+
+> Note - In many cases, the platform team will provide each developer with a unique development environment. For purposes of this guide, we will assume that the development environment is shared and will be used by multiple developers.
```

Comment:
Note - The platform team may support ephemeral development environments rather than the static environment mentioned here. For simplicity in this use case, we will assume a shared, static development environment.